scripts/write_hooks_md.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import ast
+from pathlib import Path
+from typing import Dict, Iterable
+
+HOOKS_DIR = "hooks"
+EXCLUDE_FILES = ["__init__.py", "consts.py", "helpers.py"]
+
+
+def find_hook_files() -> Iterable[Path]:
+    hooks_dir = Path(HOOKS_DIR)
+
+    for filename in hooks_dir.glob("*.py"):
+        if filename.name not in EXCLUDE_FILES:
+            yield filename
+
+
+def get_hook_docstrings() -> Dict[str, str]:
+    docstrings = {}
+
+    for hook_file in find_hook_files():
+        with hook_file.open("r") as fd:
```

Comment:
`fd` 를 통상 file descriptor (int, `os.open` 의 결과) 를 의미합니다.
```suggestion
        with hook_file.open("r") as f:
```

Reply:
`f`와 `fd`는 같은 의미인줄 알았는데 감사합니다 :)

Reply:
https://github.com/yogiyo/pre-commit-hooks/pull/3/commits/8cf0bcf3ebf1dc1eb3e9bc36ab7fedacc6dd1fcc 수정하였습니다 ~